### ISSUE TITLE
url in queryString does not redirect to specified path

### ISSUE THREAD

**Issue by jokaorgua** [harmbandstra/swagger-ui-bundle#28]
For some reason, the UI stopped redirecting direct URLs to specified yaml files

I have http://localhost/v1/doc/?url=/docs/v1/api.yaml URL. When I put this in browser I get the default swagger file https://petstore.swagger.io/v2/swagger.json

Only when I enter /docs/v1/api.yaml in the top search box I get my yaml presented in UI

Could you advise what is going on ?

I'm using version 5.0.0

**Comment by harmbandstra** [comment]
I can reproduce the issue. I'll investigate and push a fix asap.

**Comment by harmbandstra** [comment]
It was caused by a backward incompatible change in swager-ui, pushed as a patch 👎 

See: https://github.com/swagger-api/swagger-ui/releases/tag/v4.1.3

I've pinned the dependency for now. Should be fixed if you bump to `5.0.1`.

**Comment by vasilvestre** [comment]
If this isn't working for anyone, here's how I found a workaround.
```json
{
  "extra": {
    "patches": {
      "swagger-api/swagger-ui": [
        "patches/swagger-api-swagger-ui-queryconfigenabled.patch"
      ]
    }  
  }
}
```

```patch
--- a/dist/swagger-initializer.js
+++ b/dist/swagger-initializer.js
@@ -3,8 +3,9 @@
   
   // the following lines will be replaced by docker/configurator, when it runs in a docker-container
   window.ui = SwaggerUIBundle({
-    url: "https://petstore.swagger.io/v2/swagger.json",
+    url: "/doc/url.yml",
     dom_id: '#swagger-ui',
     deepLinking: true,
+    queryConfigEnabled: true,
     presets: [
       SwaggerUIBundle.presets.apis,
       SwaggerUIStandalonePreset

```

Only the line `queryConfigEnabled` is required.

Cycle : composer install deps -> patch swagger -> this bundle copy swagger files
